tests/v0_tests/test_add_documents.py

```diff
@@ -551,6 +609,30 @@ def test_add_docs_logs_deprecation_warning_if_non_tensor_fields(self):
         non_tensor_fields = ['text']
 
         with self.assertLogs('marqo', level='WARNING') as cm:
-            test_index_name = self.create_test_index(self.generic_test_index_name)
+            test_index_name = self.create_test_index(
+                cloud_test_index_to_use=CloudTestIndex.basic_index,
+                open_source_test_index_name=self.generic_test_index_name,
+            )
             self.client.index(test_index_name).add_documents(documents=documents, non_tensor_fields=non_tensor_fields)
             self.assertTrue({'`non_tensor_fields`', 'Marqo', '2.0.0.'}.issubset(set(cm.output[0].split(" "))))
+
+    def test_add_documents_mock_overwrites_mock_for_cloud_tests(self):
```

Comment:
Can you please add a docstring for this test?

Reply:
What about checking if regular mock activities work?

E.g.: does `mock_add_documents.assert_called_once` work?

Reply:
Also, if this is a test specifically to guarantee that the add_documents mocking structure works (which is a really smart idea to have!) let's move it into a new test file. For example, `test_cloud_integration_tests.py`